ProjectLighthouse/Database.cs

```diff
@@ -358,6 +360,27 @@ public async Task<(User, GameToken)?> UserAndGameTokenFromRequest(HttpRequest re
 
     #endregion
 
+    #region Password Reset Token
+
+    public User? UserFromPasswordResetToken(string pwToken)
```

Comment:
This method should be asynchronous I feel, also maybe rename this argument to `resetToken`?

Reply:
Ok, have done now